Server/tests/controllers/authController.test.js

```
@@ -303,6 +305,97 @@ describe("Auth Controller - loginUser", () => {
+		sinon.stub(jwt, "verify");
+
+		issueTokenStub = sinon.stub().returns("new_auth_token");
+		sinon.replace({ issueToken }, "issueToken", issueTokenStub);
+	});
+
+	afterEach(() => {
+		sinon.restore();
+	});
+
+	it("should reject if no refresh token is provided", async () => {
+		delete req.headers["x-refresh-token"];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)